WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -116,7 +116,7 @@ def post_announcement
     end
     title = "#{comp.name} on #{date_range_str} in #{comp.cityName}, #{comp.countryId}"
 
-    body = "The [#{comp.name}](#{root_url}results/c.php?i=#{comp.id})"
+    body = "The [#{comp.name}](#{root_url(locale:nil)}results/c.php?i=#{comp.id})"
```

Comment:
If I understand this correctly, this is a place where someone viewing the website in French would end up getting linked to a page in the default locale, yes?